.github/workflows/build.yml

```diff
@@ -55,6 +55,14 @@ jobs:
         run: |
           npm run build
 
+      - name: Diff Tokens
+        id: diff-tokens
+        run: |
+          curl https://leo.bravesoftware.com/css/variables.css > ./tokens/css/variables.old.css
+          # Note: Diff has exit code 1 for differences, and there are always
+          # some differences because the file contains a timestamp.
+          diff -u ./tokens/css/variables.old.css ./tokens/css/variables.css > ./tokens/css/variables.diff || echo "Diff Completed"
```

Comment:
```suggestion
          diff -u ./tokens/css/variables.old.css ./tokens/css/variables.css > ./tokens/css/variables.diff || :
```

Reply:
If you just want to silence `diff`, I think `|| :` makes the intent clearer

Reply:
That is exactly what I wanted to do but my bash-fu is weak :laughing: Thanks @kim0